app/models/billing_contract.rb

```diff
@@ -0,0 +1,15 @@
+class BillingContract < ApplicationRecord
+  validates :contract_num, presence: true, length: { maximum: 50 }
+  validates :is_active, inclusion: { in: [true, false] }
```

Comment:
i think it should work without `inclusion validation`, presence: true should be enough, but I can mistake

Reply:
Validation `presence: true` doesn't allow value `false`

Reply:
good